### PR TITLE
Tweets feed overflowing fixed + made circular and responsive.

### LINKED ISSUE #817: Twitter feed on the footer is overflowing
#### Description

https://user-images.githubusercontent.com/51183483/186899420-79336125-e29d-4cb4-ab62-f60765988a12.mov

#### Expected Behavior

Fix the height of the twitter feed. Reference: https://smp-spec.io
#### Enviroment:
 - OS: Mac 
 - Browser: Safari
 - Version: 
 - Device: Desktop

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by legendarykamal**
Signed-off-by: Kamal Nayan <95926324+legendarykamal@users.noreply.github.com>

**Description**

This PR fixes #817 

Tweets overflowing fixed at the footer  + made circular and responsive .

**Notes for Reviewers**
Screenshot from 2022-08-27 00-48-57

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
This feed is looking better… thanks to a sharp-dressed individual.

**Comment by Rohith-Raju** [comment]
LGTM @legendarykamal good stuff!!

**Comment by legendarykamal** [comment]
> This feed is looking better… thanks to a sharp-dressed individual.

Thank you ... @leecalcote  I was also happy  to find myself in the feeds :smiley:

**Review by asubedy — APPROVED** [review]
LGTM @legendarykamal 🚀  Thank you .

**Comment by asubedy** [comment]
@legendarykamal , there are some issues on the responsiveness, when only the twitter feed is shifted below it should be centered
Eg: 
image

**Review by Rohith-Raju — APPROVED** [review]

**Comment by legendarykamal** [comment]
@asubedy for smaller screens, It automatically comes down with the social .

image
image

**Comment by legendarykamal** [comment]
> One last merge conflict.

Resolved :)

File: _includes/footer.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 					{% endfor %}
 					<ul class="feed">
 						<div class="col l4">
-							<a class="twitter-timeline" data-height="310" data-width="310" style="border-radius: 6px;"
+							<a class="twitter-timeline" data-width="320" data-height="320"
```
Comment (leecalcote) [inline _includes/footer.html:19]: The border radius should not be removed.

What's the behavioral difference that we're seeking to effect by reducing the size of the embed?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 1 above)
Comment (legendarykamal) [reply _includes/footer.html:19]: actually :sweat_smile: they should be removed as they are overwritting already made css which is leading to this , the current design 

image

**Comment by legendarykamal** [comment]
In short this now tries to solve #838

**Comment by leecalcote** [comment]
If there's still a need for this change, will you please add it to the Websites agenda? If not, perhaps, close it?

**Comment by legendarykamal** [comment]
> 

yes will add it 👍

**Comment by legendarykamal** [comment]
> If there's still a need for this change, will you please add it to the Websites agenda? If not, perhaps, close it?

yes will add it 👍

**Comment by adithyaakrishna** [comment]
@legendarykamal Any updates on this?

**Comment by legendarykamal** [comment]
@adithyaakrishna let me wrap this up asap.

**Comment by legendarykamal** [comment]
@leecalcote , @adithyaakrishna  the new update , it looks great awaiting feedback.

image

**Review by leecalcote — APPROVED** [review]
Works in Safari

**Pull request merged**